backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -48,7 +53,19 @@ public UsersController(WebClientService webClientService, @Value("${config.vanit
         search.ifPresent(searchValue -> queryParams.add("search", searchValue));
         username.ifPresent(usernameValue -> queryParams.add("username", usernameValue));
 
-        return webClientService.getUsers(queryParams);
+        final ResponseEntity<Object> users = webClientService.getUsers(queryParams);
+        final ResponseEntity<Object> filteredUsers;
```

Comment:
I know opinions on this vary, but we're aren't using the `final` keyword in the project _except for class fields_. It ends up cluttering up the code, as most variables end up being `final`.